tests/ztest/zexpect/CMakeLists.txt

```diff
@@ -10,12 +10,5 @@ else()
   find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
   project(base)
 
-  if(CONFIG_CPLUSPLUS)
-    message(STATUS "adding main.cpp")
-    target_sources(app PRIVATE src/main.cpp)
-  else()
-    target_sources(app PRIVATE src/main.c)
-
-    target_sources_ifdef(CONFIG_USERSPACE app PRIVATE src/main_userspace.c)
```

Comment:
Looks like this is a bug of mine for forgetting to add `src/main_userspace.c` This is fine to delete, but I'll make a TODO for adding a ZTEST_USER example to `main.c` if `CONFIG_USERSPACE` is enabled. To affirm this same macro works in userspace tests.

Reply:
I've updated